docs/README.md

```diff
@@ -67,6 +67,7 @@ All blocks exported by `store-link` share the same props:
 | `displayMode` | `enum` | How the link should be displayed. Possible values are: `anchor` (displays a normal link with no styles) or `button` (displays a button that can be customized using the `buttonProps` prop.  | `anchor` |
 | `buttonProps` | `object` | How the link button should be displayed. Use this prop only when the `displayMode` prop is set as `button`. | `{ variant: primary, size: regular }` |
 | `escapeLinkRegex`   | `string` | RegExp, with global match, used to remove special characters within product specifications. (E.g. if you want to use `/[%]/g` then `escapeLinkRegex` = `[%]` )         | `undefined`   |
+| `rel` | `string` | This prop specifies the relationship between the current document and the linked document (for better SEO). This prop works the same way as the `rel` attribute from [HTML `<a>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) (HTML *anchor* element). | `undefined` |
```

Comment:
This is already a great description! How about adding this link too? https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types, it could be right after the current description, something like "Supported values can be found here", with this link.

Reply:
Awesome, @Sbodazo, and great point @victorhmp! I would suggest rephrasing the sentence, so we don't need to repeat `HTML`, for example: 
```suggestion
| `rel` | `string` | This prop specifies the relationship between the current document and the linked document for better SEO. This prop works the same way as the `rel` attribute from `<a>`, the [HTML anchor element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). | `undefined` |
```
What do you think?